packages/url/src/filter-url-for-display.js

```diff
@@ -53,3 +53,14 @@ export function filterURLForDisplay( url, maxLength = null ) {
 		truncatedFile
 	);
 }
+
+/**
+ * Removes the protocol from a URL.
+ *
+ * @param {string} url - The URL to remove the protocol from.
+ *
+ * @return {string} The URL without the protocol.
+ */
+export function removeProtocol( url ) {
+	return url.replace( /^(?:https?:)\/\/(?:www\.)?/, '' );
```

Comment:
This won't work for protocols different than HTTP/HTTPS, like `tel:` or `file:`.